### PR TITLE
02987

### PR DESCRIPTION
**Pull request by warrencj**
Changed a couple data types on older things as well as added series type.

### CODE REVIEW

File: athlinks-schema.ttl [hunk 1]
```diff
@@ -66,13 +73,22 @@ athlinks:AthleticRace
...
+  rdfs:hasProperty athlinks:volunteer ;
 
 athlinks:AthleticEvent
   a owl:Class ;
   rdfs:comment "An Athletic Event, a group of Athletic Races that are all part of the same event." ;
   rdfs:label "Athletic Race" ;
-  rdfs:subClassOf schema:SportsEvent .
+  rdfs:subClassOf schema:SportsEvent ;
+  rdfs:hasProperty athlinks:details  ;
+  rdfs:hasProperty athlinks:description ;
```
Comment (Astn) [inline athlinks-schema.ttl]: I don't think we need athlinks:details, or athlinks:description. rdfs:comment and rdfs:label should be sufficient.

File: athlinks-schema.ttl [hunk 2]
```diff
@@ -123,12 +139,12 @@ athlinks:ranking a owl:ObjectProperty;
...
   a owl:ObjectProperty ;
   rdfs:domain athlinks:TimingInterval ;
-  rdfs:range schema:QuantitativeValue .
+  rdfs:range schema:Duration .
 
 athlinks:distanceFromStart
   a owl:ObjectProperty ;
   rdfs:domain athlinks:TimingInterval ;
-  rdfs:range schema:QuantitativeValue .
+  rdfs:range schema:Duration .
```
Comment (Astn) [inline athlinks-schema.ttl]: Not sure if I follow on how a distance can be stored in a schema:Duration.

File: athlinks-schema.ttl [hunk 3]
```diff
@@ -123,12 +139,12 @@ athlinks:ranking a owl:ObjectProperty;
 athlinks:defaultPace
   a owl:ObjectProperty ;
   rdfs:domain athlinks:TimingInterval ;
-  rdfs:range schema:QuantitativeValue .
+  rdfs:range schema:Duration .
```
Comment (Astn) [inline athlinks-schema.ttl]: If the pace is per/mile or per/k... then what? I think QuantitativeValue is supposed to solve that problem.

File: athlinks-schema.ttl [hunk 4]
```diff
@@ -145,10 +161,48 @@ athlinks:bib
 
 athlinks:pace
   a owl:DataTypeProperty ;
-  rdfs:range schema:DateTime .
+  rdfs:range schema:Duration .
```
Comment (Astn) [inline athlinks-schema.ttl]: See comments on :defaultPace

File: athlinks-schema.ttl [hunk 5]
```diff
@@ -145,10 +161,48 @@ athlinks:bib
...
+  rdfs:range [ a xsd:string ;
+								owl:oneOf("Master", "Social", "Points", "Series")
+	     				] .
+
+# For  AthleticRace PROPERTIES
+athlinks:giveaway
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:require_phone
```
Comment (Astn) [inline athlinks-schema.ttl]: these "require_*" seem more like registration configurations then Race configurations.

File: athlinks-schema.ttl [hunk 6]
```diff
@@ -145,10 +161,48 @@ athlinks:bib
...
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:require_dob
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:volunteer
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
```
Comment (Astn) [inline athlinks-schema.ttl]: I think we need some better context for volunteer.

File: athlinks-schema.ttl [hunk 7]
```diff
@@ -145,10 +161,48 @@ athlinks:bib
...
+
+For AthleticEvent PROPERTIES
+athlinks:details
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:description
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
```
Comment (Astn) [inline athlinks-schema.ttl]: Details and Description feel like duplicates of RDFS:label and RDFS:Comment.

File: athlinks-schema.ttl [hunk 8]
```diff
@@ -145,10 +161,48 @@ athlinks:bib
...
+
+athlinks:isPartnerRace
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:externalResultsUrl
```
Comment (Astn) [inline athlinks-schema.ttl]: If we want our links to get this specific.. they should be extending MainEntityOfPage. But even in that case.. you could achieve the same thing, but using MainEnityOfPage for the externalraceUrl. and then on the "results' object use mainEntityOfpage to get the externalResultsUrl.

**Review by Astn — CHANGES_REQUESTED** [review]
Think we got some fixing before we merge this one.

File: athlinks-schema.ttl (hunk 2 above)
Comment (warrencj) [reply athlinks-schema.ttl]: Whoops, my fault...got a little overzealous with the ctrl-v

File: athlinks-schema.ttl (hunk 4 above)
Comment (warrencj) [reply athlinks-schema.ttl]: When I looked up pace, I saw durations rather than km/min or whatever

Comment (warrencj) [reply athlinks-schema.ttl]: then this one should be QuantitativeValue as well, right?

File: athlinks-schema.ttl (hunk 5 above)
Comment (warrencj) [reply athlinks-schema.ttl]: Those seem to be part of the race in most systems that we have looked at BUT if we create something that holds the registration separate from the race then I think I see what you are saying.

File: athlinks-schema.ttl (hunk 6 above)
Comment (warrencj) [reply athlinks-schema.ttl]: Agreed

File: athlinks-schema.ttl (hunk 1 above)
Comment (PetarPropulsionApp) [reply athlinks-schema.ttl]: agree

File: athlinks-schema.ttl (hunk 3 above)
Comment (PetarPropulsionApp) [reply athlinks-schema.ttl]: I didn't add changes for that fields
I created fork 
https://github.com/athlinks/rdf-ontology/pull/5/commits/cdcbae5a715ce84deff65688519c7cc5e6ac27ef
you will see my changes

File: athlinks-schema.ttl (hunk 4 above)
Comment (Astn) [reply athlinks-schema.ttl]: I think so.. but I'm guessing maybe there is a quantitativeDuration??

File: athlinks-schema.ttl (hunk 5 above)
Comment (Astn) [reply athlinks-schema.ttl]: ya, I'm just figuring when we start to map Ctlive data... we are going to have tons of Registration configuration stuff...
And we likely will want to keep that separate from the Race just due to quantity / complexity of the registration configuration.

Comment (PetarPropulsionApp) [reply athlinks-schema.ttl]: agree

File: athlinks-schema.ttl [hunk 9]
```diff
@@ -14,6 +14,13 @@
 
 ### CLASSES
 
+athlinks:User
+  a rdfs:Class ;
+  rdfs:subClassOf schema:Person;
+  rdfs:comment "User .";
+  rdfs:label "User".
+  rdfs:hasProperty athlinks:id
```
Comment (Astn) [inline athlinks-schema.ttl]: I think we should remove athlinks:User until we figure out how to store social accounts.

File: athlinks-schema.ttl [hunk 10]
```diff
@@ -66,13 +73,20 @@ athlinks:AthleticRace
...
 
 athlinks:AthleticEvent
   a owl:Class ;
   rdfs:comment "An Athletic Event, a group of Athletic Races that are all part of the same event." ;
   rdfs:label "Athletic Race" ;
-  rdfs:subClassOf schema:SportsEvent .
+  rdfs:subClassOf schema:SportsEvent ;
+  rdfs:hasProperty athlinks:isPartnerRace ;
+  rdfs:hasProperty athlinks:externalResultsUrl ;
+  rdfs:hasProperty athlinks:externalRaceUrl .
```
Comment (Astn) [inline athlinks-schema.ttl]: These externalRaceUrl / externalResultsUrl should be removed.

File: athlinks-schema.ttl [hunk 11]
```diff
@@ -66,13 +73,20 @@ athlinks:AthleticRace
...
+  rdfs:hasProperty athlinks:require_dob ;
+  rdfs:hasProperty athlinks:volunteer ;
 
 athlinks:AthleticEvent
   a owl:Class ;
   rdfs:comment "An Athletic Event, a group of Athletic Races that are all part of the same event." ;
   rdfs:label "Athletic Race" ;
-  rdfs:subClassOf schema:SportsEvent .
+  rdfs:subClassOf schema:SportsEvent ;
+  rdfs:hasProperty athlinks:isPartnerRace ;
```
Comment (Astn) [inline athlinks-schema.ttl]: I don't grok this isPartnerRace.. lets pull it until it can be better defined.

File: athlinks-schema.ttl [hunk 12]
```diff
@@ -66,13 +73,20 @@ athlinks:AthleticRace
   rdfs:comment "An athletic race, a distinct course." ;
   rdfs:label "Athletic Race" ;
   rdfs:hasProperty athlinks:raceType ;
-  rdfs:subClassOf schema:SportsEvent .
+  rdfs:subClassOf schema:SportsEvent ;
+  rdfs:hasProperty athlinks:giveaway  ;
+  rdfs:hasProperty athlinks:require_phone ;
+  rdfs:hasProperty athlinks:require_dob ;
+  rdfs:hasProperty athlinks:volunteer ;
```
Comment (Astn) [inline athlinks-schema.ttl]: all 4 of these... (require_phone, require_dob, volunteer, giveaway) seem out of context.. I'm guessing the should be on the registration setup, or part of collecting registration data? I just don't even know how we would use these properties on the Race... unless volunteer has a range of Person? But what does require_phone mean on a race?

**Review by Astn — CHANGES_REQUESTED** [review]

File: athlinks-schema.ttl (hunk 9 above)
Comment (warrencj) [reply athlinks-schema.ttl]: Removed

File: athlinks-schema.ttl [hunk 13]
```diff
@@ -145,10 +145,10 @@ athlinks:bib
 
 athlinks:pace
   a owl:DataTypeProperty ;
-  rdfs:range schema:DateTime .
+  rdfs:range schema:QuantitativeValue .
```
Comment (Astn) [inline athlinks-schema.ttl:148]: I think we need to solve the ambiguity around athlinks:pace. Not sure the solution yet, but this is too ambiguous as it is.

**Review by Astn — CHANGES_REQUESTED** [review]

**Pull request merged**